### PR TITLE
NSB101 replace objectives with paragraphs and time estimates

### PR DESCRIPTION
**Pull request by DavidBoike**
Related to call between Udi/Britt/Kyle/myself.

@kbaley please review

### CODE REVIEW

**Comment by DavidBoike** [comment]
@kbaley has left for holidays so won't be able to review. Goal was to make it the lessons less formal and more approachable, and to make a specific promise to the reader to get them to agree to just that lesson. Removing WIP.

File: tutorials/nservicebus-101/lesson-5/tutorial.md [hunk 1]
```diff
@@ -7,15 +7,7 @@ In software systems, exceptions will occur. Even with perfect, bug-free code, pr
...
-
-## Objectives
-
-By the end of this lesson, you will have learned:
-
- * The different causes of exceptions, and how to deal with them
- * How to use immediate retries to resolve transient errors
- * How to use delayed retries to resolve semi-transient errors
- * How to use message replay to resolve systemic errors
+By the end of this lesson (approximately 20-25 minutes) you will have learend the different causes of errors, how to use immediate retries to resolve transient errors, how to use delayed retries to resolve semi-transient errors, and how to use message replay to resolve systemic errors.
```
Comment (kbaley) [inline tutorials/nservicebus-101/lesson-5/tutorial.md]: Similar to the last comment:

> By the end of this lesson (approximately 20-25 minutes) you will learn the different causes of errors and techniques we can use to address them.

File: tutorials/nservicebus-101/lesson-4/tutorial.md [hunk 2]
```diff
@@ -5,18 +5,7 @@ reviewed: 2016-11-16
...
-## Objectives
-
-By the end of this lesson, you will have learned:
-
- * The difference between commands and events
- * How the Publish/Subscribe pattern helps to create more maintainable code
- * How to define and name events
- * How to publish an event
- * How to subscribe to an event
+In this lesson (approximately 25-30 minutes) you'll learn the differences between commands and events, how the Publish/Subscribe pattern helps us to create more maintainable code, and how to define, publish, and subscribe to an event.
```
Comment (kbaley) [inline tutorials/nservicebus-101/lesson-4/tutorial.md]: This still reads like a list that's trying not to be one. Maybe we don't have to be so prescriptive:

> In this lesson (approximately 25-30 minutes) you'll learn how the publish/subscribe pattern can help you create more maintainable code. Through this pattern we'll define, publish, and subscribe to an event.

**Comment by weralabaj** [comment]
@kbaley @DavidBoike not sure hwat's the status of this PR? if you need maintainers review, please ping me on Slack.

**Comment by DavidBoike** [comment]
@kbaley check out the diff https://github.com/Particular/docs.particular.net/pull/2334/commits/461d0835c6593ea03422e55f6c5790f2482e3532 and see if that sounds better.

@weralabaj Sorry about that. I'll set it to WIP for now and ping when Kyle approves.

**Comment by kbaley** [comment]
@DavidBoike LGTM

**Comment by DavidBoike** [comment]
@weralabaj take a look?

**Pull request merged**